fonts/fonts.css

```diff
@@ -0,0 +1,26 @@
+@font-face {
+	font-family: 'montserrat';
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/GurevAleksey_itogovaya/pullRequest?prid=6087836)